src/Microsoft.PowerShell.Commands.Utility/commands/utility/ConsoleColorCmdlet.cs

```diff
@@ -89,6 +89,10 @@ public ConsoleColorCmdlet()
         }
 
         #region helper
+        [System.Diagnostics.CodeAnalysis.SuppressMessage(
+            "Usage",
+            "CA2208:Instantiate argument exceptions correctly",
+            Justification = "https://github.com/PowerShell/PowerShell/issues/13909")]
```

Comment:
Please use local suppression. It is difficult to find that we suppress.

Reply:
`BuildOutOfRangeErrorRecord` is a three line method, why do we need local suppression here?

Reply:
There are other cases below.